yarn.lock

```diff
@@ -476,6 +496,25 @@
     "@aws-cdk/cx-api" "1.124.0"
     constructs "^3.3.69"
 
+"@aws-cdk/aws-apigateway@1.131.0", "@aws-cdk/aws-apigateway@~1.131.0":
+  version "1.131.0"
+  resolved "https://registry.yarnpkg.com/@aws-cdk/aws-apigateway/-/aws-apigateway-1.131.0.tgz#03275ac707f3250bc0b317ad62b448586f083ecc"
+  integrity sha512-0kt7ZLibQO4UG4jNpVvL4BJFkWIw2+A4H7H+d9PF/Evi//aTXQ6vR3P+tMZAg9S+3ll85KlfZFgqY6MHdliKJA==
+  dependencies:
+    "@aws-cdk/aws-certificatemanager" "1.131.0"
+    "@aws-cdk/aws-cloudwatch" "1.131.0"
+    "@aws-cdk/aws-cognito" "1.131.0"
+    "@aws-cdk/aws-ec2" "1.131.0"
+    "@aws-cdk/aws-elasticloadbalancingv2" "1.131.0"
+    "@aws-cdk/aws-iam" "1.131.0"
+    "@aws-cdk/aws-lambda" "1.131.0"
+    "@aws-cdk/aws-logs" "1.131.0"
+    "@aws-cdk/aws-s3" "1.131.0"
+    "@aws-cdk/aws-s3-assets" "1.131.0"
+    "@aws-cdk/core" "1.131.0"
+    "@aws-cdk/cx-api" "1.131.0"
+    constructs "^3.3.69"
+
 "@aws-cdk/aws-apigatewayv2@~1.124.0":
```

Comment:
I'm seeing `~1.124.0` references here and throughout the lockfile

Reply:
That's because published versions are the same as we've currently in package.json and monorepo seems to be not smart enought with yarn 1.x ? it is pulling in those deps from published npm...